src/components/about.css

```diff
@@ -0,0 +1,31 @@
+h3 {
```

Comment:
This isn't a good idea because general naming like this can override styles in the app and lead to unexpected behaviour